Discourage Combat Secretaries

juxjux9930 opened the pull request:
🆑
tweak: Changes the executive assistant smartgun to a compact, removes body armor from Executive Assistant lockers, and adjusts their required/maximum skills.
/🆑

PsiOmegaDelta commented on maps/torch/job/corporate_jobs.dm:
```diff
@@ -53,11 +53,11 @@
 	                    SKILL_EVA         = SKILL_BASIC,
 	                    SKILL_COMBAT      = SKILL_BASIC,
-	                    SKILL_WEAPONS     = SKILL_ADEPT,
+	                    SKILL_WEAPONS     = SKILL_Basic,
```
The non-uppercased `Basic` will likely fail to compile

juxjux9930 replied:
Fixed.

Spookerton commented on maps/torch/structures/closets/misc.dm:
```diff
@@ -67,11 +67,10 @@
 		/obj/item/weapon/gun/energy/gun/secure/corporate,
```
You left this one in. :B

juxjux9930 replied:
Nobody said I was smart. Should be fixed, thanks.

juxjux9930 committed:
Tweaks combat-encouraging items & skills from Executive Assistant

Removes the body armor in the locker & changes the smartgun to a compact. Reduces required combat skills and maximum skills.
Update corporate_jobs.dm
Update misc.dm
